cmd/notary/keys.go

```diff
@@ -408,8 +415,8 @@ func (k *keyCommander) importKeys(cmd *cobra.Command, args []string) error {
 	for _, file := range args {
 		from, err := os.OpenFile(file, os.O_RDONLY, notary.PrivKeyPerms)
 		defer from.Close()
-
-		if err = utils.ImportKeys(from, importers); err != nil {
+		passRetreiver := k.getRetriever()
```

Comment:
Non-blocking nit - can we change this param name to "passRetriever"  ("retriever" is misspelled :))